src/utils/load_game/load_game.cpp

```diff
@@ -0,0 +1,124 @@
+#include "load_game.h"
+#include "../../creature/main_character/main_character.h"
+#include "../utils.h"
+#include <iostream>
+#include <fstream>
+#include <string>
+
+// struct GameData
+// {
+// 	int hp;
+// 	int ac;
+// 	int speed;
+// 	int strength;
+// 	int dexterity;
+// 	int constitution;
+// 	int intelligence;
+// 	int wisdom;
+// 	int charisma;
+// };
+
+void LoadSavedGame(MainCharacter& main_character)
+{
+	std::ifstream in_file("savefile.txt");
+	if (in_file.is_open())
+	{
+		std::string line;
+		int lineCount = 0;
+		int game_data[] = {-1, -1, -1, -1, -1, -1, -1, -1, -1};
```

Comment:
Perhaps we should consider splitting `game_data` to `mc_hp, mc_ac, mc_speed, mc_ability_scores[6]` . I believe it would make the code a bit more clearer.